source/_integrations/drop_connect.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: DROP
+description: Instructions on how to integrate DROP into Home Assistant.
+ha_category:
+  - Sensor
+ha_config_flow: true
+ha_codeowners:
+  - '@ChandlerSystems'
+  - '@pfrazer'
+ha_domain: drop_connect
+ha_iot_class: Local Push
+ha_mqtt: true
+ha_release: '2024.1'
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The DROP integration provides connectivity with 
+[DROP smart water system](https://dropconnect.com) devices via a local MQTT API.
+
+### Supported devices
+
+There is currently support for the following DROP products within Home Assistant:
+
+- **DROP Hub**: centralized system status and control.
+- **Leak Detector**: reports detected leaks around the home.
+- **Softener**: improves water quality by decreasing water hardness with integral water shutoff.
+- **Filter**: improves water quality by removing contaminants with integral water shutoff.
+- **Protection Valve**: automatic water shutoff with slow leak detection.
+- **Pump Controller**: smart replacement for an FSG pressure switch.
+- **RO Filter**: reverse osmosis drinking water filtration.
+- **Salt Sensor**: alerts when the salt level in the softener brine tank is low.
+
+### Configuration
+
+To use DROP with Home Assistant, you must have already configured the [MQTT](/integrations/mqtt/) platform.
+
+To enable the DROP integration, use the DROP Connect app to connect your DROP Hub to your MQTT broker:
+- Start the DROP Connect app and connect to your DROP Hub.
+- In the 'Hub Network Settings' section on the System > Advanced page, tap the 'Configure MQTT' button.
+- Enter the MQTT broker address, port number, username, and password.
```

Comment:
May be you could explain on how to configure a username and password on the Mosquitto Addon as the default username and password on the MQTT broker are set up automatically and are not known by the user.

![afbeelding](https://github.com/home-assistant/home-assistant.io/assets/7188918/53b62daa-5edb-4e6c-90ff-09f8bb5f8ece)

Reply:
Additional instructions added in c5bbd4bdfa87654a60b0854d14adeaf3eb80a2e9.